bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingConfigDescriptionProvider.java

```diff
@@ -57,8 +57,8 @@ public ConfigDescription getConfigDescription(URI uri, Locale locale) {
         }
 
         // First, get the thing type so we get the generic config descriptions
-        ThingUID thingUID = new ThingUID(uri.getSchemeSpecificPart());
-        ThingType thingType = thingTypeRegistry.getThingType(thingUID.getThingTypeUID());
+        ThingTypeUID thingTypeUID = new ThingTypeUID(uri.getSchemeSpecificPart());
```

Comment:
Why is `uri.getSchemeSpecificPart()` all of a sudden a thing type UID and not anymore a thing UID? Note that we are within the ThingConfigDescriptionProvider, which deals with uris of Things.